doc/README.md

```diff
@@ -0,0 +1,21 @@
+# pyglmnet documentation
+
+See full documentation page [here](http://pavanramkumar.github.io/pyglmnet/).
+
+We use `sphinx` to generate documentation page.
+You can install dependencies for documentation page by running
+
+```bash
+$ pip install -r requirements.txt
+```
+
+To build documentation page, run `make html`. All static files will be built in
```

Comment:
I was able to see the files without running the http server ... you just need to open `_build/html/index.html`